### PR TITLE
small additions to manta

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by hexylena** [comment]
pinging @drosofff since we'd love to have this tool on UseGalaxy.eu :)

**Comment by drosofff** [comment]
Nice ! Thanks @hexylena and @bgruening 

@bgruening could you bump the wapper version to `+galaxy8` please ?

File: tools/manta/manta.xml [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
     <description>Manta calls structural variants (SVs) and indels from mapped paired-end sequencing reads.</description>
-
     <macros>
         <import>manta_macros.xml</import>
     </macros>
     <expand macro="requirements"/>
     <expand macro="stdio"/>
 
     <command detect_errors="exit_code"><![CDATA[
-    @VERSION@
```
Comment (bgruening) [inline tools/manta/manta.xml:12]: was that needed @drosofff ?

Comment (drosofff) [reply tools/manta/manta.xml:12]: I don't think so ! I am ok with removing it

**Comment by drosofff** [comment]
Thanks @bgruening !

**Pull request merged**